features/buyer/buyer_journey.feature

```diff
@@ -0,0 +1,26 @@
+@not-production @functional-test
+Feature: Buyer user journey through Digital Marketplace
+
+Scenario: Check the admin frontend /_status
+  Given I am logged in as a admin user
```

Comment:
would "I am logged in as **an** admin user" also work? It would be more grammatical this way.